docs/best_practices.md

```diff
@@ -0,0 +1,130 @@
+<!--[metadata]>
++++
+title = "Best Practices for Using Notary"
+description = "A set of recommended practices for key management, delegating trust, bootstrapping repos, and more."
+keywords = ["docker, Notary, notary-client, docker content trust, content trust", "best practices", "recommended use"]
+[menu.main]
+parent="mn_notary"
+weight=1
++++
+<![end-metadata]-->
+
+# Best Practices for Using Docker Notary
+
+This document describes good and recommended practices for using the Notary client.
+It includes recommendations on key management, bootstrapping trust for your repositories,
+delegating trust to other signers. You may wish to refer to the [Getting Started](getting_started.md)
+and [Advanced Usage](advanced_usage.md) documents for more information on some of the
```

Comment:
might be jumping the gun here, but we should link to the `command_reference.md` doc if/when that's merged